lib/oxidized/model/comware.rb

```diff
@@ -42,6 +42,10 @@ class Comware < Oxidized::Model
     end
 
     post_login 'screen-length disable'
+    # optionally you can add into system-view:
```

Comment:
The screen-length disable command is run all the time anyway so I'm not sure those notes are useful.